hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -177,7 +177,14 @@
     { name:    "Stub",
       type:    "bit",
       default: "0",
-      desc:    "Stub out the core of entropy_src logic"
+      desc:    "Stub out the USB device logic, leaving only the register interface"
+      local:   "false",
+      expose:  "true"
+    },
+    { name:    "AlertSkewCycles",
```

Comment:
This is a topgen handled parameter. Topgen connects this paramter to `top_pkg::AlertSkewCycles`.

Reply:
Okay, thanks, I was unaware of that.